### PR TITLE
Labels edition for legends

### LINKED ISSUE #10687: Be able to override legend labels
Designs in https://github.com/CartoDB/design/issues/284

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes #10687 on Builder's side.

custom-labels

CR frontend part @xavijam 
CR backend legend definition @gfiorav

### CODE REVIEW

File: lib/formats/legends/custom_choropleth.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
   "properties": {
     "prefix": { "type": "string" },
     "suffix": { "type": "string" },
+    "left_label": { "type": "string" },
+    "right_label": { "type": "string" },
```
Comment (gfiorav) [inline lib/formats/legends/custom_choropleth.json]: Could you explain what the difference between a normal choropleth and a custom choropleth is? There's a lot of duplication so maybe we could rethink it

Comment (nobuti) [inline lib/formats/legends/custom_choropleth.json]: custom choropleths is for pixel aggregated map. The right legend must be a choropleth but the maps api doesn't return the values we need to render it. So we fake it. A little more context https://github.com/CartoDB/cartodb/issues/10224

**Comment by nobuti** [comment]
Tested in ded04

**Comment by gfiorav** [comment]
🇪🇸 for the backend side!

**Comment by nobuti** [comment]
Shrinkwrap brings some updates in deep-insights and torque.js, could you take a look @donflopez to make sure all is ok? cc @xavijam

**Comment by xavijam** [comment]
Same in https://github.com/CartoDB/cartodb/pull/10787.

**Comment by donflopez** [comment]
🇪🇸 torque side

**Pull request merged**